### PR TITLE
Remove spaces from filepaths

### PR DESCRIPTION
**Pull request by dostrander**
**Why?**

Using runfiles in Bazel causes issues when spaces are introduced in file paths (see https://github.com/bazelbuild/bazel/issues/4327). This is a work around until this bug is fixed. 

Let me know if there are any issues around this.

### CODE REVIEW

**Review by justinseanmartin — APPROVED** [review]

**Comment by justinseanmartin** [comment]
Will wait for feedback from @NickEntin and @dfed before merging this one. The patch release to the older `3.x` release felt safer though, given that I expect most consumers have already moved forward to 4.x versions.

**Review by NickEntin — APPROVED** [review]
This looks like it should be a safe change for a patch release 👍

**Comment by dfed** [comment]
Seems reasonable. This'll potentially cause problems for folk who have hard coded source paths... but that's not an explicitly supported use-case so I think we're okay here.

**Pull request merged**
